pkg/util/ipvs/ipvs_linux.go

```diff
@@ -214,10 +214,15 @@ func toVirtualServer(svc *ipvs.Service) (*VirtualServer, error) {
 		Port:      svc.Port,
 		Scheduler: svc.SchedName,
 		Protocol:  protocolNumbeToString(ProtoType(svc.Protocol)),
-		Flags:     ServiceFlags(svc.Flags),
 		Timeout:   svc.Timeout,
 	}
 
+	// Test flags >= 0x2
+	if svc.Flags&FlagHashed == 0 {
+		return nil, fmt.Errorf("Flags of successfully created IPVS service should be >= %d since every service is hashed into the service table", FlagHashed)
+	}
+	vs.Flags = ServiceFlags(svc.Flags - uint32(FlagHashed))
```

Comment:
I would say &^, buut OK

Reply:
I understand your meaning, since the valid value range is [2, 3] for now, I think it's fine to use bit operator `&^`. Then, we need to update test cases. BTW, I would like to add some comments...